module_9/task4.php

```diff
@@ -0,0 +1,23 @@
+<pre>
+<?php
+
+$city1 = 250;
+$city1Radius = 300;
+$city2 = 750;
+$city2Radius = 300;
+
+$countOfCars = 10;//Количество машин
+
+for ($i = 0; $i < $countOfCars; $i++) {
```

Comment:
Please create an array with all cars and their positions before making a solution